### PR TITLE
frontend: styled keystoreconnectpromp component

### PR DESCRIPTION
**Pull request by shonsirsha**
[Preview]:

dikx

qdwasc

### CODE REVIEW

File: frontends/web/src/components/keystoreconnectprompt.tsx [hunk 1]
```diff
@@ -18,40 +18,68 @@ import { useTranslation } from 'react-i18next';
...
-    // TODO: make this look nice.
     return (
-      <>
-        { data.keystoreName === '' ?
+      <Dialog title="Connect BitBox02" medium open>
+        <p className={styles.text}>{ data.keystoreName === '' ?
           t('connectKeystore.promptNoName') :
           t('connectKeystore.promptWithName', { name: data.keystoreName })
-        }
-        {/* Software keystore is unlocked from the app, so we add the button here.
```
Comment (benma) [inline frontends/web/src/components/keystoreconnectprompt.tsx:39]: this comment line is missing below

Comment (thisconnect) [reply frontends/web/src/components/keystoreconnectprompt.tsx:39]: SkipForTesting should be conditionally rendered see:

https://github.com/digitalbitbox/bitbox-wallet-app/blob/b3f968702a030a55eca5eadb1582de1ae8848b5a/frontends/web/src/routes/device/waiting.tsx#L32

it's a bit ugly, maybe you could refactor that SkipForTesting does this logic internally?

**Review by thisconnect — CHANGES_REQUESTED** [review]

**Comment by thisconnect** [comment]
Oh nice, I checked the wrong branch and  didn't look at skipfortesting in this branch.

File: frontends/web/src/components/keystoreconnectprompt.tsx [hunk 2]
```diff
@@ -18,40 +18,68 @@ import { useTranslation } from 'react-i18next';
...
-            Te BitBox01 is ignored - BitBox01 users will simply need to unlock before being prompted.
-          */}
-        <SkipForTesting />
-        <Button primary onClick={() => cancelConnectKeystore()}>{t('dialog.cancel')}</Button>
-      </>
+        }.
+        </p>
+        <div className={styles.bitboxContainer}>
+          {/*
+        /* Software keystore is unlocked from the app, so we add the SkipForTesting button here (only for development).
```
Comment (thisconnect) [inline frontends/web/src/components/keystoreconnectprompt.tsx:46]: nit: double begin of comment `/* /*`

**Review by thisconnect — APPROVED** [review]
LGTM with nit

**Pull request merged**